release-notes/aws/v9.0.1.md

```diff
@@ -0,0 +1,56 @@
+## :zap: Giant Swarm Release 9.0.1 for AWS is now active for you! :zap:
+
+This release includes some reliability and user experience improvements.
+
+Below, you can find more details on components that were changed with this release.
+
+### Kubernetes [v1.15.11](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.15.md#changelog-since-v11510)
+- Updated from v1.15.5.
+- Fixed a regression where the kubelet would fail to update the ready status of pods.
+- Various other improvements to the management of cloud and cluster resources.
+
+### aws-operator v5.5.1
+- Streamlined image templating for core components for quicker and easier releases in the future.
+- Enabled per-cluster configuration of kube-proxy's `conntrackMaxPerCore` parameter.
+
+### chart-operator [v0.12.1](https://github.com/giantswarm/chart-operator/releases/tag/v0.12.1)
+
+- Added chartmigration resource for migrating from chartconfig to chart CRs.
+- Updated release resource.
+  - Do not wait when installing or updating long running Helm releases.
+  - Use version field from chart CR to reduce number of HTTP requests to pull chart tarballs.
+  - Wait for deleted Helm release before removing finalizer. 
+- Updated status resource.
+  - Improve reason field in CR status when installing a chart fails.
+- Removed legacy chartconfig controller.
+
+### cluster-operator [v0.23.8](https://github.com/giantswarm/cluster-operator/releases/tag/v0.23.8)
+
+- Stop setting IC replicas count.
+- Re-enable proxy protocol by default for AWS clusters.
+- Fix cluster deletion by gracefully handling Tenant Cluster API errors.
+- Support classifying clusters into profiles based on available cluster resources.
+
+### coredns v1.6.5 ([Giant Swarm app v1.1.8](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v118-2020-03-20))
+
+- Updated coredns to upstream version 1.6.5.
+- Removed CPU limits.
+- Updated manifests for Kubernetes 1.16 compatibility. 
+- Make `autopath` plugin configurable, optional and disabled by default.
+- Allow custom forward configuration destination and options.
+- Add Pod Disruption Budget.
+
+### kube-state-metrics v1.9.5 ([Giant Swarm app v1.0.5](https://github.com/giantswarm/kube-state-metrics-app/blob/master/CHANGELOG.md#v105))
+
+- Updated from `kube-state-metrics` v1.9.2 for some stability fixes.
+- Removed addon resizer sidecar container.
+- Removed CPU and memory limits.
+
+### nginx-ingress-controller v0.30.0 ([Giant Swarm app v1.6.4](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v164-2020-03-17))
+
+- Enabled HorizontalPodAutoscaler by default, accompanied with following configuration defaults changes:
+  - CPU resource requests changed from 500m to 2 CPU
+  - Memory requests changed from 600Mi to 2.5GB
+  - Number of NGINX worker processes changed from 4 to 1
```

Comment:
```suggestion
### nginx-ingress-controller v0.30.0 ([Giant Swarm app v1.6.5](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v165-2020-03-23))

- Enabled HorizontalPodAutoscaler by default for selected cluster profiles.
```

This aligns with details communicated in kvm releases notes https://github.com/giantswarm/releases/blob/master/release-notes/kvm/v11.2.1.md